api/v1alpha3/awscluster_types.go

```diff
@@ -131,6 +131,11 @@ type AWSLoadBalancerSpec struct {
 	// Subnets sets the subnets that should be applied to the control plane load balancer (defaults to discovered subnets for managed VPCs or an empty set for unmanaged VPCs)
 	// +optional
 	Subnets []string `json:"subnets,omitempty"`
+
+	// SecurityGroups sets the security groups used by the load balancer. Expected to be security group IDs.
+	// This is optional - if not provided new security groups will be created for the load balancer
+	// +optional
+	SecurityGroups []string `json:"securityGroups,omitempty"`
```

Comment:
It seems an `apiserver-lb` role security group is [still required](https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/1db3792ca47657849e96fc27ffafd7d178535be7/pkg/cloud/services/securitygroup/securitygroups.go#L216) in the securityGroupOverrides if overrides are meant to be used. Would the user just repeat one of the loadbalancer sg IDs in there?

Reply:
Yeah I can imagine a user might do that. I need to take another look at the implementation but this might be better named `AdditionalSecurityGroups` which would be added to any overrides provided in the other section.

Reply:
The logic for this has changed. The field is now `AdditionalSecurityGroups` and appends sgs, so repeating one of the loadbalancer sg IDs is no longer necessary.